lookout/style/format/__main__.py

```diff
@@ -68,6 +69,22 @@ def add_parser(name, help):
                              help="Programming language to use.")
     rule_parser.add_argument("-m", "--model-path", required=True,
                              help="Path to saved FormatModel.")
+
+    # Style robustness quality report
+    robust_parser = add_parser("robust", "Quality report made by analyzing how well the model is"
```

Comment:
Please change "robust" to "eval" because that's exactly what we do.

Reply:
"eval" is already a command according to Egor's initial quality report